src/ClientNet462/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Newtonsoft.Json" version="4.0.1" targetFramework="net462" />
```

Comment:
4.0.1 ? So old?

Reply:
The sample was requested for this specific version.  The code works with no changes with Newtonsoft.Json 13.0.1 also.

Reply:
I think that was a mistake. I see 12.0.3 in zipped sample. 4.0.1 is a version of `GraphQL.Client` 😉

Reply:
Well, no matter.  This sample works with basically any version of Newtonsoft.Json.

Reply:
OK